Vagrantfile

```diff
@@ -38,7 +38,9 @@ $defaults = {
   set_hostname: false,
   ssh_port_start: 10000,
   ssh: {
-    user: 'vagrant'
+    user: 'vagrant',
+    password: nil,
+    private_key: nil
```

Comment:
The above two lines are not required. Did you put it here just for the case of documenting these options?

Reply:
You are right. There is no need for these.